templates/ts-apollo-mongodb-backend/src/custom-resolvers.ts

```diff
@@ -0,0 +1,25 @@
+import { GraphbackContext, GraphbackCRUDService, Maybe, Scalars } from "graphback";
+
+/**
+ * Types can be generated using GraphQL-Code-Generator
+ */
+export type Note = {
+  id?: Maybe<Scalars['String']>;
+  title?: Maybe<Scalars['String']>;
+  description?: Maybe<Scalars['String']>;
+};
+
+const getNoteService = (context: GraphbackContext) => {
+  return context.graphback?.services?.Note as GraphbackCRUDService<Note>
+}
+
+export const customResolvers = {
+  Query: {
+    printNote: async (parent: any, variables: any, context: GraphbackContext, info: any) => {
+      const noteService = getNoteService(context);
+      const notes = await noteService.findBy({}, { graphback: {} });
```

Comment:
```suggestion
      const notes = await noteService.findBy({}, context);
```

Reply:
Try doing it :) It will fail with `graphback is undefined`

Reply:
Hmm something fishy might be going on then. Let me re-check. We've it here https://github.com/aerogear/OpenVolunteerPlatform/blob/master/platform/server/src/resolvers/custom-resolvers.ts#L25 without any issue.